packages/content-conformance/src/rules/top-of-file-h1.js

```diff
@@ -0,0 +1,30 @@
+// eslint-disable-next-line import/no-anonymous-default-export
+/** @type {import('../types.js').ConformanceRuleBase} */
+export default {
+  type: 'content',
+  id: 'top-of-file-h1',
+  description: 'Each content file must contain an H1 the top.',
+  executor: {
+    async contentFile(file, context) {
```

Comment:
@nayres We should also ignore partials for this rule. This can be accomplished by checking `file.isPartial`

Reply:
Done!